### ISSUE TITLE
Older incompatible liquid-fire version installed by default with newer Ember versions

### ISSUE THREAD

**Issue by bitwolfe** [ember-learn/ember-cli-addon-docs#440]
When running AddonDocs on a newer Ember versions (3.15 in my case) I had to use a yarn resolution to get the newer liquid-fire version that is compatible. By default it installs the 0.29.x version which causes the following error:

```
loader.js:247 Uncaught Error: Could not find module `liquid-fire/templates/version-specific/get-outlet-state` imported from `liquid-fire/ember-internals/version-specific/index`
```

I worked around it with Yarn by adding the following resolution to my package.json:

```
  "resolutions": {
    "ember-cli-addon-docs/liquid-fire": "^0.31.0"
  },
```

**Comment by rwwagner90** [comment]
We could probably just bump to the latest version. @samselikoff do you know of a reason we cannot?

**Comment by SergeAstapov** [comment]
Looks like it was fixed in #414
